commit d57cd2570ef5580168b99d61fe6ce90ce879bac5
Author: Christophe Giboudeaux <christophe@krop.fr>
Date:   Sat Apr 13 12:20:28 2019 +0200

    Fix the qmake command and the include subdir for openSUSE
    
diff --git a/bin/fixqt4headers.pl b/bin/fixqt4headers.pl
index b572f469dc..25478a03ac 100755
--- a/bin/fixqt4headers.pl
+++ b/bin/fixqt4headers.pl
@@ -139,14 +139,14 @@ sub findQtHeaders
 # -------- MAIN
 
 if ($qtdir) {
-    $qtIncludeDir = $qtdir . '/include';
+    $qtIncludeDir = $qtdir . '/include/qt5';
 } else {
-    $qtIncludeDir = `qmake -query QT_INSTALL_HEADERS`;
+    $qtIncludeDir = `qmake-qt5 -query QT_INSTALL_HEADERS`;
     chop($qtIncludeDir);
 }
 
 die "The location of the Qt 5 include files could not be determined.\n"
-        ."Please ensure qmake can be found in PATH or pass the command line option --qtdir.\n"
+        ."Please ensure qmake-qt5 can be found in PATH or pass the command line option --qtdir.\n"
     unless -d $qtIncludeDir;
 
 findQtHeaders('QtCore', $qtIncludeDir);
@@ -154,7 +154,7 @@ findQtHeaders('QtConcurrent', $qtIncludeDir);
 findQtHeaders('QtWidgets', $qtIncludeDir);
 findQtHeaders('QtPrintSupport', $qtIncludeDir);
 
-if (-d $qtIncludeDir . '/include/QtMultimedia') {
+if (-d $qtIncludeDir . '/include/qt5/QtMultimedia') {
     findQtHeaders('QtMultimedia', $qtIncludeDir);
     findQtHeaders('QtMultimediaWidgets', $qtIncludeDir);
 } elsif (-d $qtIncludeDir . '/../qtmultimedia' ) {
@@ -164,7 +164,7 @@ if (-d $qtIncludeDir . '/include/QtMultimedia') {
 }
 
 # Support porting from "Qt 4.99" QtDeclarative to QtQuick (QQuickItem et al)
-if (-d $qtIncludeDir . '/include/QtQuick') {
+if (-d $qtIncludeDir . '/include/qt5/QtQuick') {
     findQtHeaders('QtQuick', $qtIncludeDir);
 } elsif (-d $qtIncludeDir . '/../qtdeclarative' ) {
     # This is the case if QTDIR points to a source tree instead of an installed Qt
